packages/composite-checkout/src/components/checkout-review-order.js

```diff
@@ -17,6 +17,7 @@ import {
 
 export default function CheckoutReviewOrder( { summary, className } ) {
 	const [ items, total ] = useLineItems();
+
 	if ( summary ) {
```

Comment:
In line with my other comment, we should probably remove this `summary` block since it's no longer possible after https://github.com/Automattic/wp-calypso/pull/37261/files#diff-5b63d2418405e222d04228a455428b98L252 unless we want to add that back in.